deb/openmediavault/etc/udev/rules.d/99-openmediavault-dev-disk-by-id.rules

```diff
@@ -33,6 +33,16 @@ ACTION=="add", KERNEL=="sd*", SUBSYSTEMS=="usb", \
   ENV{ID_SERIAL}="USB-%c-$env{ID_INSTANCE}", \
   ENV{ID_SERIAL_SHORT}="%c"
 
+# JMicron JMS561U two ports SATA 6Gb/s bridge
```

Comment:
Do you have a link to the device?

Reply:
https://www.sabrent.com/product/EC-DFLT/usb-3-0-sata-external-hard-drive-lay-flat-docking-station-2-5-3-5in-hdd-ssd/